.travis.yml

```diff
@@ -26,3 +47,5 @@ script:
   - make
   - if [ $USE_CMAKE == "YES" ] ; then CTEST_OUTPUT_ON_FAILURE=1 make test ; fi
   - if [ $USE_CMAKE == "NO" ] ; then make check ; fi
+  - cppcheck --language=c --std=c89 --verbose --quiet --error-exitcode=1 --enable=warning,performance,portability src/
```

Comment:
It seems that whatever version of cppcheck is installed does not like --language=c flag. You might need to experiment to see what version of cppcheck is available on Travis-CI and only use the flags which are available at that version.